packages/calcite-components/src/components/input-date-picker/assets/t9n/messages.json

```diff
@@ -1,4 +1,4 @@
 {
   "chooseDate": "Choose date",
-  "dateFormat": "Date Format:"
+  "dateFormat": "Date Format: {format}"
```

Comment:
We could avoid the `{format}` literal in messages and do below in JSX. Thoughts?

`{messages.dateFormat}  {this.localeData?.placeholder)}`

Reply:
I think we need it in the template so translations can position accordingly.